src/websockets/ohlc_handler.js

```diff
@@ -30,6 +30,11 @@ define(['websockets/binary_websockets','charts/chartingRequestMap','jquery','jqu
     liveapi.events.on('candles', function (data) {
         var key = (data.echo_req.ticks_history + data.echo_req.granularity).toUpperCase();
         for ( var index in data.candles ) {
+            //This is for microsoft products. 
+            //They some how also include the include() while iterating over the object.
+            if(index === 'includes'){
```

Comment:
@4p00rv I am not sure if I like this, but can we do it in a better way? That is just do `(data.candles || []).forEach()`

My recommendation is we should use ES6/ES7 wherever possible so that we are improving code gradually.